MetabolicAtlas/MetabolicAtlas
# feat: citation widgets

**Related issue(s) and PR(s)**  
This PR closes #907 and is created by @e0 and @inghylt 



**Type of change**  

 
**List of changes made**  

- New Vue component CitationWidget.vue created and added to the Citation.vue component
- Reload functionality added for routes (required to make the widgets work)
- Section added to Privacy Policy

**Screenshot of the fix**  

![image](https://user-images.githubusercontent.com/32480198/159286647-66da69a3-1765-410c-ba7f-e0ed3ca68508.png)


**Testing**  
- https://dev-metatlas.csbi.chalmers.se/about/citation Recommended to test on the dev server so you can see how this will look once deployed as the reload is faster there
- Go to About -> Citation
- Go to https://dev-metatlas.csbi.chalmers.se/about/citation directly
- Hover the different widgets
- Look at the page in different screen sizes
- Read the extended privacy policy

**Further comments**  

Reload is required for the widgets to be displayed when you visit the page via About -> Citation. A number of other different solution was explored, none else was however successful. Although it feels sub optimal to modify App.vue for a specific and small page, the added functionality is reusable and does not affect the overall performance.

**Definition of Done checklist**
